phylogenetic/rules/prepare_sequences.smk

```diff
@@ -0,0 +1,157 @@
+"""
+This part of the workflow prepares sequences for constructing the
+phylogenetic tree.
+"""
+
+# FIXME once the ingest build is finalized and data is uploaded to S3,
+# this rule should be uncommented, and the location of metadata.tvs
+# and sequences.fasta in various rules should be adjusted.
+# rule download:
+#     output:
+#         metadata="data/metadata.tsv.zst",
+#         sequences="data/sequences.fasta.zst",
+#     params:
+#         sequences_url="https://data.nextstrain.org/files/workflows/yellow-fever/sequences.fasta.zst",
+#         metadata_url="https://data.nextstrain.org/files/workflows/yellow-fever/metadata.tsv.zst",
+#     shell:
+#         r"""
+#         curl -fsSL --compressed {params.sequences_url:q} --output {output.sequences}
+#         curl -fsSL --compressed {params.metadata_url:q} --output {output.metadata}
+#         """
+
+
+# FIXME once the ingest build is finalized and data is uploaded to S3,
+# this rule should be uncommented, and the location of metadata.tvs
+# and sequences.fasta in various rules should be adjusted.
+# rule decompress:
+#     input:
+#         sequences="data/sequences.fasta.zst",
+#         metadata="data/metadata.tsv.zst",
+#     output:
+#         sequences="data/sequences.fasta",
+#         metadata="data/metadata.tsv",
+#     shell:
+#         r"""
+#         zstd -d -c {input.sequences} > {output.sequences}
+#         zstd -d -c {input.metadata} > {output.metadata}
+#         """
+
+
+rule filter_genome:
+    input:
+        exclude = config["files"]["genome"]["exclude"],
+        include = config["files"]["genome"]["include"],
+        #FIXME metadata = "data/metadata.tsv",
+        #FIXME sequences = "data/sequences.fasta"
+        metadata = "../ingest/results/metadata.tsv",
+        sequences = "../ingest/results/sequences.fasta"
```

Comment:
In order for this to run in the Nextstrain docker runtime, you need to add the top level `nextstrain-pathogen.yaml` file.

Reply:
added, thanks.